src/defaults.ts

```diff
@@ -0,0 +1,759 @@
+import {
```

Comment:
These defaults are going to balloon the bundle. Why is this the right place to be placing the defaults?

Reply:
@sethyates Good point. Sam and I are thinking the best solution is to have a separate `ketch-defaults` repo that is imported by figurehead and lanyard. That way other things that use `ketch-types` won't see their bundle sizes balloon